virttest/guest_agent.py

```diff
@@ -678,22 +678,38 @@ def verify_fsfreeze_status(self, expected):
             raise VAgentFreezeStatusError(self.vm.name, status, expected)
 
     @error_context.context_aware
-    def fsfreeze(self, check_status=True, timeout=FSFREEZE_TIMEOUT):
+    def fsfreeze(self, check_status=True, timeout=FSFREEZE_TIMEOUT,
+                 fsfreeze_list=False, mountpoints=None):
         """
-        Freeze File system on guest.
+        Freeze File system on guest, there are two commands,
+        "guest-fsfreeze-freeze" and "guest-fsfreeze-freeze-list".
+        guest-fsfreeze-freeze: Sync and freeze all freezable,
+        local guest filesystems.
+        guest-fsfreeze-freeze-list: Sync and freeze specified guest
+        filesystems
 
         :param check_status: Force this function to check the fsfreeze status
                              before/after sending cmd.
+        :param fsfreeze_list: Freeze fs with guest-fsfreeze-freeze or
+                              guest-fsfreeze-freeze-list.
```

Comment:
`fsfreeze_list` should apply to `guest-fsfreeze-freeze-list` only, not  `guest-fsfreeze-freeze`, right?

Reply:
yes, if fsfreeze_list is False, then use guest-fsfreeze-freeze command, else use guest-fsfreeze-freeze-list command.

Reply:
@xiagao I mean the description might confuse users: `Freeze fs with guest-fsfreeze-freeze or guest-fsfreeze-freeze-list.`

Reply:
I update just now ,could you check if it's better?